test/rule-matches/aria-form-field-label-matches.js

```diff
@@ -0,0 +1,91 @@
+describe('aria-form-field-label-matches', function() {
+	'use strict';
+
+	var fixture = document.getElementById('fixture');
+	var queryFixture = axe.testUtils.queryFixture;
+	var rule = axe._audit.rules.find(function(rule) {
+		return rule.id === 'aria-form-field-label';
+	});
+
+	afterEach(function() {
+		fixture.innerHTML = '';
+	});
+
+	it('returns false when node has no role', function() {
+		var vNode = queryFixture('<textarea id="target" title="Label"></textarea>');
```

Comment:
Missing a role, such as `role="textbox"`. This element doesn't even match the selector.